src/main/java/org/curtinfrc/frc2025/Constants.java

```diff
@@ -24,6 +31,8 @@
  */
 public final class Constants {
   private static final RobotType robotType = RobotType.SIMBOT;
+  public static final double DIMENSIONS_X = 550; // mm
+  public static final double DIMENSIONS_Y = 570;
```

Comment:
These are robot dimensions right?
```suggestion
  public static final double ROBOT_X = 550; // mm
  public static final double ROBOT_Y = 570;
```